### ISSUE TITLE
screenshots and support for cypress-cucumber-preprocessor

### ISSUE THREAD

**Issue by swatinigam** [boxblinkracer/cypress-testrail#13]
Hello @boxblinkracer, firstly, just want to say thank you for this plugin. 🙏  I've been trying it out with my project that uses Cypress v10.8.0 and it seems to be working just fine so far.  I have two questions, I would have asked them in the Discussions tab but I don't see it so asking them in this issue instead.

Q1.  When my test fails, I see its status as failed in testrail. But I don't see the screenshot anywhere. Where in the test run/ test case result would it be, if it were to exist in TestRail? 
here is my Cypress.env.json file :
```
{
    "testrail": {
      "domain": "xxx.testrail.io/",
      "username": "xyz",
      "password": "xyz",
      "projectId": "1",
      "suiteId": "2",
      "runId": "4",
      "screenshots": true
    }
  }
```

I can see the cypress/screenshots folder populate with the screenshot when the test fails but I am not seeing the screenshot in TestRail. 

Q2. Can this package also be used for Cypress projects that use the Cypress-cucumber preprocessor? i.e. https://www.npmjs.com/package/cypress-cucumber-preprocessor? If yes, could you please share how to supply the testIDs for something like Scenario Outlines which have multiple tests within them? 

Thank you for your help!

**Comment by boxblinkracer** [comment]
Hi

first of all thank you so much for your kind words
great that you like it

Q1)
so the screenshot should usually be inside a result (please see my screenshot attached).
in your case i think the configuration might have both modes...so I guess runId is set, so inside run R4 in the result of the testcase.

<img width="909" alt="Screenshot_2023-01-05_at_21_23_27" src="https://user-images.githubusercontent.com/5579326/210873292-470ca695-6aab-4092-ae3f-ba861408e9ea.png">

Q2) i have no idea :D 
i didnt use it yet
so the integration basically extracts the name of the test....so if cucumber is wrapping cypress tests then in the core a test with a name and therefore a testcase id should exist

ill create another ticket for it, but also, i'm a bit out of time at the moment
so i cannot guarantee a fast implementation :)

**Comment by mturcan** [comment]
subscribe to the @swatinigam 's  Q2 . any information regarding cucumber scenarios and testrail integration would be much appreciated :pray:

**Comment by boxblinkracer** [comment]
Hi there
sorry for not replying earlier (no worries...still mainained, just released v2.6)

i have never used it so far, so I need to dig deeper (but im totally open to support it)
if anyone has a plug and play template that could help me to get started, feel free to share it :)

**Comment by boxblinkracer** [comment]
so, this is finally done
i will continue in the other ticket and close this one

https://github.com/boxblinkracer/cypress-testrail/issues/14
